### DISCUSSION TITLE
not exploit but cool thing I found

### DISCUSSION THREAD

**Discussion by Perplexedykwhat** [3kh0/ext-remover#1390] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
I found how to clear cookies with an extension. its called web developer or something, it allows you to clear or disable cookies on a web page. it can also grab html I think?

**Comment by Charlieis1nsan3** [comment]
you could also clear extensions when you click that (i)  icon and click delete cookies. and for html'. just use a html' fetcher.(website)

**Reply by Perplexedykwhat** [reply]
cool
